token-repository/src/main/scala/io/renku/tokenrepository/repository/refresh/EventsFinder.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.tokenrepository.repository
+package refresh
+
+import AccessTokenCrypto.EncryptedAccessToken
+import ProjectsTokensDB.SessionResource
+import association.Project
+import cats.effect.MonadCancelThrow
+import io.renku.db.{DbClient, SqlStatement}
+import io.renku.graph.model.projects
+import io.renku.metrics.LabeledHistogram
+
+import java.time.LocalDate
+
+private trait EventsFinder[F[_]] {
```

Comment:
Probably miss something: I don't understand why it is a "events finder" when it queries project tokens?

Reply:
True, I renamed it to `TokensToRefreshFinder`